en/docs/Administer/ProductConfigurations/changing-the-default-transport.md

```diff
@@ -1,34 +1,29 @@
 # Changing the Default Transport
 
-On the back end, APIs are Apache Synapse configurations that WSO2 API Manager accesses through a transport. The default API Manager transport is the PassThrough transport, but you can configure a different default transport in your `axis2.xml` file. For example, to use the HTTP-NIO transport as the default, go to the `<APIM_HOME>/repository/conf/axis2` folder, open the `axis2.xml` file, and then in the "Transport Ins" and "Transport Outs" sections, comment out the PassThrough configurations and uncomment the configurations for the HTTP-NIO transport.
+On the back end, APIs are Apache Synapse configurations that WSO2 API Manager accesses through a transport. The default API Manager transport is the PassThrough transport, but you can configure a different default transport in your `deployment.toml` file.
 
 !!! info
-WSO2 products do not use the HTTP/S servlet transport configurations that are in `axis2.xml` file. Instead, they use Tomcat-level servlet transports, which are used by the management console in `<PRODUCT_HOME>/repository/conf/tomcat/catalina-server.xml` file .
+    WSO2 products do not use the HTTP/S servlet transport configurations that are in `axis2.xml` file. Instead, they use Tomcat-level servlet transports, which are used by the management console in `<PRODUCT_HOME>/repository/conf/tomcat/catalina-server.xml` file .
 
 
 The following topics provide more information on these transports:
 
--   [HTTP PassThrough transport](#ChangingtheDefaultTransport-HTTPPassThroughtransport)
--   [HTTP-NIO transport](#ChangingtheDefaultTransport-HTTP-NIOtransport)
--   [Transport receiver parameters](#ChangingtheDefaultTransport-Transportreceiverparameters)
--   [Transport sender parameters](#ChangingtheDefaultTransport-Transportsenderparameters)
--   [Connection throttling](#ChangingtheDefaultTransport-Connectionthrottling)
+-   [HTTP PassThrough Transport](#http-passthrough-transport)
+-   [Transport Receiver Parameters](#transport-receiver-parameters)
+-   [Transport Sender Parameters](#transport-sender-parameters)
+-   [Connection Throttling](#connection-throttling)
 
-### HTTP PassThrough transport
+### HTTP PassThrough Transport
 
 HTTP PassThrough Transport is the default, non-blocking HTTP transport implementation based on HTTP Core NIO and is specially designed for streaming messages. It is similar to the old message relay transport, but it does not care about the content type and simply streams all received messages through. It also has a simpler and cleaner model for forwarding messages back and forth. The two classes that implement the receiver and sender APIs are `org.apache.synapse.transport.passthru.PassThroughHttpListener` and `org.apache.synapse.transport.passthru.PassThroughHttpSender` , respectively. The PassThrough Transport does not require the binary relay builder and expanding formatter.
```

Comment:
Shall we change [1] to [2].

[1] It is similar to the old message relay transport, but it does not care about the content type and simply streams all received messages through.

[2] It is similar to the old message relay transport, but it does not take the content type into consideration and simply streams all the messages that it receives.

Reply:
fixed with https://github.com/wso2/docs-apim/pull/540/commits/66d4d47c3b663dccc4ce708bc0ae2f9cf1e0ddff